### PR TITLE
chore(docs): Add a goproxy settings description in docs/kitex and docs/hertz

### PR DESCRIPTION
**Pull request by zhu-mi-shan**
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
在docs/kitex中，和docs/hertz中添加设置代理的说明。

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/en/docs/hertz/getting-started/_index.md [hunk 1]
```diff
@@ -93,7 +93,8 @@ First, you need to install the command tool hz which is used in this chapter:
 1. Confirm the `GOPATH` environment has been defined correctly (For example `export GOPATH=~/go`)
    and the `$GOPATH/bin` has been added to `PATH` environment (For example `export PATH=$GOPATH/bin:$PATH`);
    Attention, do not set `GOPATH` to a directory that the current user does not have read/write access to.
-2. Install hz: `go install github.com/cloudwego/hertz/cmd/hz@latest`.
+2. You may need to set up a domestic proxy: `go env -w GOPROXY=https://goproxy.cn`.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/getting-started/_index.md:96]: For English speakers, they don't need a domestic proxy

File: content/en/docs/kitex/Getting started/prerequisite.md [hunk 2]
```diff
@@ -39,6 +39,12 @@ The code generation tools used in Kitex include the IDL compiler and the kitex t
 
 The IDL compiler can parse IDL and generate corresponding serialization and deserialization code. Kitex supports two types of IDL, Thrift and protobuf, which rely on thriftgo and protoc, respectively, for parsing.
 
+First you may need to set up a domestic proxy:
+
+```bash
+go env -w GOPROXY=https://goproxy.cn
+```
+
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Getting started/prerequisite.md:47]: Not needed for English

File: layouts/community/list.html [hunk 3]
```diff
@@ -29,4 +29,4 @@ <h1>{{ .Title }}</h1>
 	{{ end }}
 	{{ partial "page-meta-lastmod.html" . }}
 </div>
-{{ end }}
+{{ end }}
```
Comment (GuangmingLuo) [inline layouts/community/list.html:32]: do not modify this file if there is no change.

File: content/en/docs/hertz/getting-started/_index.md (hunk 1 above)
Comment (zhu-mi-shan) [reply content/en/docs/hertz/getting-started/_index.md]: I have deleted the English part, thanks for the suggestion！

File: content/en/docs/kitex/Getting started/prerequisite.md (hunk 2 above)
Comment (zhu-mi-shan) [reply content/en/docs/kitex/Getting started/prerequisite.md]: I have deleted the English part, thanks for the suggestion！

File: layouts/community/list.html (hunk 3 above)
Comment (zhu-mi-shan) [reply layouts/community/list.html]: I have deleted the commit, thanks for the suggestion！

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**